docs/helpers/adoc-generator.go.tmpl

```diff
@@ -84,10 +103,33 @@ func GetAnnotatedVariables(s interface{}) []ConfigField {
 		default:
 			desc := field.Tag.Get("desc")
 			env, ok := field.Tag.Lookup("env")
+			isDeprecated := false
 			if !ok {
 				continue
 			}
-			v := fmt.Sprintf("%v", value.Interface())			
+			deprecationVersion, ok := field.Tag.Lookup("deprecationVersion")
+			if !ok {
+				deprecationVersion = ""
+			}
```

Comment:
Is this necessary? I'd expect `deprecationVersion` to be empty if `ok == false`?